langkit/documentation.py

```diff
@@ -743,9 +743,11 @@ def instantiate_templates(doc_dict):
         Set the root node for the unit Handle to Root. This unties the previous
         root handle. If Root is not No_Node_Rewriting_Handle, this also ties
         Root to Handle.
+        Root must not already be tied to another analysis unit handle.
```

Comment:
Please add an empty line to separate paragraphs (otherwise paragraphs will be merged during template rendering). Here and below, whenever you want to create a new paragraph.